content/docs/rfcs/20/README.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: 20
+title: 20/ETH-DM
+name: Ethereum Direct Message
+status: raw
+editor: Franck Royer <franck@status.im>
+contributors:
+---
+
+This specification explains the Ethereum Direct Message protocol
+which enables a peer to send a direct message to another peer
+using the Waku v2 network, and the peer's Ethereum address.
+
+# Goal
+
+Alice wants to send an encrypted message to Bob, where only Bob can decrypt the message.
+
+# Variables
+
+Here are the variables used in the protocol and their definition:
+
+- `A` is Alice's Ethereum root HD public key (also named account, address),
+- `B` is Bob's Ethereum root HD public key (also named account, address),
+- `a` is the private key of `A`, and is only known by Alice,
+- `b` is the private key of `B`, and is only known by Bob.
+
+# Design Requirements
+
+The proposed protocol MUST adhere to the following design requirements:
+
+1. Alice knows Bob's Ethereum root HD public key `B`,
+2. Alice wants to send message `M` to Bob,
+3. Bob SHOULD be able to get `M` using [13/WAKU2-STORE](/spec/13), when querying a store node that hosts `M`,
+4. Bob MUST recognize he is `M`'s recipient when relaying it via [11/WAKU2-RELAY](/spec/11),
+5. Carole MUST NOT be able to read `M`'s content even if she is storing it or relaying it.
+
+## Out of scope
+
+At this stage, we acknowledge that:
+
+1. Because `Bw` is part of the message in clear,
+Alice can know how many messages from other parties Bob receive,
+and Carole can see that how many messages a recipient `Bw` is receiving (unlinkability is broken).
+
+# Steps
+
+1. Alice MUST derive Bob's waku public Key `Bw` from `B`,
+2. Alice SHOULD derive her own waku public key `Aw` from `A`,
+3. Alice creates `M'` which MUST contain `M` and `Aw`, it MAY contain `A`,
+4. Alice encrypts `M'` using `Bw`, resulting in `m'`,
+5. Alice creates waku message `Mw` with
+   `payload` `m'` and
+   `contentTopic` `/waku/2/eth-dm/child-pubkey/Bw/proto`,
```

Comment:
Not sure I fully understand here having `child-pubkey` and `Bw` in this topic. Aren't they the same?
(same remark for all contentTopics below.

Reply:
`child-pubkey` is literally the string `child-pubkey`. The aim is to qualify the data used here, ie, what is `Bw`.

We know that by having `Bw` there, we are breaking unlinkability so maybe in the future we may use a different data type instead of `Bw`, let's say `foo` that permits unlinkability. In this case, the `contentTopic` would then become `/waku/2/eth-dm/qualifier-for-foo/foo`.

Please let me know if you have an idea on how I could make this clearer.